wallet-sdk/src/main/kotlin/org/stellar/walletsdk/Wallet.kt

```diff
@@ -157,10 +154,6 @@ data class ApplicationConfiguration(
     }
 }
 
-typealias Base64Decoder = ((String) -> ByteArray)
-
 internal typealias ClientConfig = HttpClientConfig<OkHttpConfig>
 
 internal typealias ClientConfigFn = (ClientConfig.() -> Unit)
-
-internal val defaultBase64Decoder: Base64Decoder = { Base64.getDecoder().decode(it) }
```

Comment:
We don't need to handle Android differently anymore?

Reply:
Yes, Kotlin now has [Base64 decoder as a part of standard library](https://kotlinlang.org/docs/whatsnew1820.html#support-for-base64-encoding)